Formula/r/rage.rb

```diff
@@ -0,0 +1,45 @@
+class Rage < Formula
+  desc "Simple, modern, secure file encryption"
+  homepage "https://str4d.xyz/rage"
+  url "https://github.com/str4d/rage/archive/refs/tags/v0.10.0.tar.gz"
+  sha256 "34c39c28f8032c144a43aea96e58159fe69526f5ff91cb813083530adcaa6ea4"
+  license any_of: ["MIT", "Apache-2.0"]
+  head "https://github.com/str4d/rage.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "./rage")
+
+    install_completions("rage")
+    install_completions("rage-keygen")
+
+    man.install Dir["target/release/manpages/*"]
+  end
+
+  def install_completions(base_name)
+    src_dir = "target/release/completions"
+
+    bash_completion.install { "#{src_dir}/#{base_name}.bash" => base_name }
+    fish_completion.install "#{src_dir}/#{base_name}.fish"
+    zsh_completion.install "#{src_dir}/_#{base_name}"
+  end
```

Comment:
Please don't use a separate function for this, since it makes the formula harder to read.